scripts/mbedtls_dev/bignum_common.py

```diff
@@ -74,6 +74,10 @@ def combination_pairs(values: List[T]) -> List[Tuple[T, T]]:
     """Return all pair combinations from input values."""
     return [(x, y) for x in values for y in values]
 
+def hex_digits_for_limb(limbs: int, bits_in_limb: int) -> int:
```

Comment:
I am curious as to what the pylint issue which needed that change was. 

Is there any specific issue thatrequires us to add an extra function to the stack and spaghettify the current logic?

It used to be: 

`self.arg_a` ->  uses the `self.hex_digits` which calls

~~~~~~
    @property
    def hex_digits(self) -> int:
        return 2 * (self.limbs * self.bits_in_limb // 8)
~~~~~~

Now it is:

`self.hex digits` and `self.arg_a` are both independantly calling `bignum_common.hex_digits` which is doing the calculation.

I have no objection with a utility function doing that calculation, but if that is genuinely needed. I do not like changing random code which we spend serious time to design, just so we can keep pylint happy.

If using a very well used class member makes it to fail, we should look at the configuration of the tool.

Reply:
The pylint issue was a line too long error. The reason for created a new function in the bignum module to keep the hex digit calculation there because this seems ugly and hard to understand:
```python
    @property
    def arg_a(self) -> str:
        return super().format_arg('{:x}'.format(self.int_a)).zfill(2 * self.hex_digits - 2 * self.bits_in_limb // 8)
```

The `arg_a` has a different size then the other parameters so overriding the `hex_digits` function is not an option.